app/actions/authActions.js

```diff
@@ -35,38 +36,53 @@ type AccountType = ?{
   signingFunction?: Function,
   isHardwareLogin: boolean,
   isWatchOnly?: boolean,
+  hasInternetConnectivity: boolean,
```

Comment:
Adding this here seemed like the best place for me as we care about performing the check for this particular flow at the point where a user signs in